### PR TITLE
Ship

### PR DESCRIPTION
**Pull request by GDemps**
Finshed Ship class with sunk? and hit methods passing all tests.

### CODE REVIEW

File: test/ship_test.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ship'
+
+class ShipTest < Minitest::Test
+
+def test_ship_exists_and_holds_attributes
```
Comment (ajtran303) [inline test/ship_test.rb:7]: Nice combo

**Review by ajtran303 — APPROVED** [review]
Looking good, nice test of assert_equal `true` and `false`

**Comment by GDemps** [comment]
Can you read this?

**Comment by ajtran303** [comment]
at 6:10 pm i rcvd ur msg "Can you read this?" in my email inbox

**Pull request merged**